src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -29,20 +30,18 @@ const DiscoverySearch = ({
   setDiscoveryLoadingData = () => {},
   searchText = '',
   setSearchText = () => {},
-  isFocus = true,
   placeholderText = StringConstant.discoverySearchBarPlaceholder,
   setIsFocus = () => {},
   setIsFirstTimeOpen = () => {},
   fetchDiscoveryData = () => {},
+  fetchData,
   onCancelToken = () => {},
   hideBackIcon = false
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The prop `fetchData` has been added, and `isFocus` has been removed. Make sure these changes are reflected wherever this component is used. Also, ensure that the new prop `fetchData` is properly handled within the component.
```diff
-  isFocus = true,
+  fetchData,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->